README.md

```diff
@@ -1 +1 @@
-# review-trials
\ No newline at end of file
+# projectWithAdama
```

Comment:
A well-written, descriptive ReadMe page introduces your project to potential employers and makes you look professional. 😎  I suggest you use  [this template](https://github.com/microverseinc/readme-template) , but remember to customize it to your project.